raven/utilities/geoserver.py

```diff
@@ -1,12 +1,11 @@
-"""
-GeoServer interaction operations.
+"""GeoServer interaction operations.
 
 Working assumptions for this module:
 * Point coordinates are passed as shapely.geometry.Point instances.
 * BBox coordinates are passed as (lon1, lat1, lon2, lat2).
 * Shapes (polygons) are passed as shapely.geometry.shape parsable objects.
 * All functions that require a CRS have a CRS argument with a default set to WGS84.
-* GEO_URL points to the GeoServer instance hosting all files.
+* GEOSERVER_URL points to the GeoServer instance hosting all files.
```

Comment:
Oh, rename of a public configuration variable name.  Is there a way to still support the old `GEO_URL` var name for backward compatibility?  Any existing deployments, unless the admin read the Changelog, will not know about this variable rename.

Breaking the public API is the main reason we often have to pin older version in our Jupyter env build while waiting for downstream package to adapt to the new API.  Let's not continue this bad habit of breaking backward compatibility.